components/next_steps_view/steps/invite_members_step/invite_members_step.tsx

```diff
@@ -250,6 +255,15 @@ class InviteMembersStep extends React.PureComponent<Props, State> {
         return `${getSiteURL()}/signup_user_complete/?id=${this.props.team.invite_id}`;
     }
 
+    handleLinkClick = async (e: React.MouseEvent<HTMLButtonElement, MouseEvent>) => {
+        e.preventDefault();
+        trackEvent('upgrade_mm_cloud', 'click_upgrade_invite_members_step');
+        this.props.actions.openModal!({
+            modalId: ModalIdentifiers.CLOUD_PURCHASE,
+            dialogType: PurchaseModal,
+        });
+    };
```

Comment:
Wondering if it makes sense to move this to the `UpgradeLink` component to avoid duplication when it's used